### PR TITLE
New method in FlutterUtils: declaresFlutterWeb()

### PR DESCRIPTION
**Pull request by jwren**
New method in FlutterUtils: this method checks for "dependencies: flutter_web" in a pubspec file.

### CODE REVIEW

**Comment by jwren** [comment]
@devoncarew

File: src/io/flutter/FlutterUtils.java [hunk 1]
```diff
@@ -260,6 +260,43 @@ public static PluginId getPluginId() {
     return pluginId;
   }
 
+  /**
+   * Returns true if passed pubspec declares a flutter or flutter_web dependency.
+   * <p>
+   * This method is provided explicitly instead of calling declaresFlutter(VirtualFile) || declaresFlutterWeb(VirtualFile) so only one read
+   * and parsing of hte pubspec is made.
+   */
+  public static boolean declaresFlutterOrFlutterWeb(@NotNull final VirtualFile pubspec) {
```
Comment (devoncarew) [inline src/io/flutter/FlutterUtils.java]: `declaresFlutterAny()`?

File: src/io/flutter/FlutterUtils.java [hunk 2]
```diff
@@ -260,6 +260,43 @@ public static PluginId getPluginId() {
     return pluginId;
   }
 
+  /**
+   * Returns true if passed pubspec declares a flutter or flutter_web dependency.
+   * <p>
+   * This method is provided explicitly instead of calling declaresFlutter(VirtualFile) || declaresFlutterWeb(VirtualFile) so only one read
+   * and parsing of hte pubspec is made.
```
Comment (devoncarew) [inline src/io/flutter/FlutterUtils.java]: typo: the

Comment (jwren) [reply src/io/flutter/FlutterUtils.java]: fixed.

File: src/io/flutter/FlutterUtils.java (hunk 1 above)
Comment (jwren) [reply src/io/flutter/FlutterUtils.java]: changing

**Comment by jwren** [comment]
Just trying to keep PRs smaller than when I went through making the Flutter module change (to the extent possible.)

**Comment by jwren** [comment]
FYI-- @jacob314

**Pull request merged**